tests/test_jax_visualizer.py

```
@@ -0,0 +1,89 @@
+    res = ProfileResult(0, 0, 0, 0, 0, [], {})
+
+    # Should not throw, just log warning
+    viz.plot_memory_timeline(res)
+
+
+@mock.patch("stormlog.jax.visualizer.plt")
+def test_plot_memory_timeline_save_path(mock_plt: Any) -> None:
+    viz = MemoryVisualizer()
+    snapshots = [MemorySnapshot(1, "s1", 100, 100, 0, 0, {})]
+    res = ProfileResult(0, 5, 200, 150, 100, snapshots, {})
+    viz.plot_memory_timeline(res, save_path="/tmp/test.png")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.png"

(S108)